circle.yml

```diff
@@ -228,7 +228,6 @@ commands:
             RUN echo "current user: $(whoami)"
             ENV CI=1
             ENV CYPRESS_INTERNAL_FORCE_SCAFFOLD=1
-            WORKDIR /app
```

Comment:
Any ideas why this was removed?

Reply:
I went ahead and added it back. I'm not sure.